annotation/tests/test_distribution.py

```diff
@@ -1277,3 +1291,278 @@ def test_get_page_number_combinations(
     expected: List[int],
 ) -> None:
     assert get_page_number_combinations(file_pages, user_pages) == expected
+
+
+@pytest.mark.parametrize(
+    (
+        "split_multipage_doc_setup",
+        "validation_type",
+        "annotators",
+        "annotation_tasks",
+        "expected_output",
+    ),
+    (
+        (
+            "",
+            ValidationSchema.cross,
+            [{"user_id": "0"}, {"user_id": "1"}],
+            [{"user_id": "0"}, {"user_id": "1"}],
+            [],
```

Comment:
you can simply specify tuple of ids and create list of dicts with user ids in the actual test, effectively making this parametrize much smaller and easier to read